slides/bitcoin-primer.tex

```diff
@@ -0,0 +1,137 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Bitcoin Primer}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Definition}
+Some key aspects of Bitcoin: \vspace{1em}
+
+	\begin{enumerate}
+		\item<2-> competitively created
+		\item<3-> virtually represented
+		\item<4-> decentrally managed
+	\end{enumerate}
+	\vspace{1em}	
+\uncover<5->{Bitcoin is a decentrally managed and unique public database.} \\	 			\vspace{1em}
+\uncover<6->{The innovation of bitcoin is the conscious ommission of central actors.} %and 		minimal dependencies on 3rd parties
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Bitcoin building blocks}
+\input{../assets/figures/bitcoin-system.tex}
+\begin{center}
+\uncover<2->{
+Because of the decentralized infrastructure, the transaction \textbf{capacity}, transaction \textbf{legitimacy} and transaction \textbf{consensus} are much more difficult to achieve.
+}
+\end{center}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction capacity}
+\textbf{Goal:} ensuring the ability to initiate a transaction  \\
+\vspace{1em}
+\uncover<2->{Bitcoin network 
+        \begin{itemize}
+		\item {consists of all users}
+		\item {primary communication path}
+		%\item<2->{completely decentral}
+		\end{itemize}
+		}
+\vspace{1em}
+\uncover<3->{Bitcoin protocol
+        \begin{itemize}
+        \item<3->{defines manner in which communication has to take place (ruleset)}
+        \end{itemize}
+        }     
+\end{frame}
+%%%
+
+
+%%%
+\begin{frame}{Transaction capacity}  %example banking vs. decentral system?
+
+{\input{../assets/figures/transaction-capacity.tex}}
+
+\uncover<2->{        
+        \begin{itemize}
+		\item<1-> permissionless, censorship-resistant
+		\item<1-> participants are equal
+		\item<1-> peer-to-peer network %(an. Bit Torrent) %robust
+		\end{itemize}
+		}
+\end{frame}
+%%%
+
+
+%%%
+\begin{frame}{Transaction legitimacy}
+\textbf{Goal:} ensuring that a transaction was initiated by the actual owner
+	\vspace{1em}
+\uncover<2->{
+	\input{../assets/figures/transaction-manipulation.tex}
+	\vspace{2em}
+\textbf{Tool:} asymmetric cryptography
+	}
+\end{frame}
+%%%
+
+
+%%%
+\begin{frame}{Blocks and the blockchain}
+Transactions are bundled into blocks \\
+\includegraphics[width=6cm]{../assets/images/block_1.png} \\
+\uncover<2->{
+Those blocks are sequentially linked $\rightarrow $ blockchain \\
+\includegraphics[width=6cm]{../assets/images/blocks_3.png}
+} 
```

Comment:
I think it would make sense to insert a slide after this to explain how the blocks are linked in more detail (e.g. with the figure of three blocks showing how each use the ID of the preceding block).

In addition to that, I think the original slides on the "bitcoin miner" were quite nice and would fit well at this stage as they contain nice visualizations and lead up to the next slide.

Reply:
@dariothuerkauf I tend to agree with @Rothj. While this slide deck serves as a high-level overview, it may still be useful to add some more context and get a little closer to the original slides.

Reply:
I agree. I will add the original slides on the "bitcoin miner" in the next commit.